docs/source/certificates/certificates.md

```diff
@@ -0,0 +1,173 @@
+
+### TLS CA Certificates
+
+| Certificate | Description | Location | Impact if expired | Reference |
+|----------------|-----|---------------------------|-----------|---|
+| Root Certificate | Certificate used to make administrative requests to the CA, such as to register new users. | Stored on disk in the CA directory. <br><br>ca-cert.pem | CA root certificates are issued for 15 years. A New certificate must be issued.  |  |
+| CA Signing Certificate | The certificate used to make administrative requests to the CA, such as to register new users. | Depends on implementation <br /><br /><code>msp <br />&nbsp;&nbsp;&nbsp;├── IssuerPublicKey <br>&nbsp;&nbsp;&nbsp;├── IssuerRevocationPublicKey <br>&nbsp;&nbsp;&nbsp;├── cacerts <br>&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;└── localhost-7053.pem <br>&nbsp;&nbsp;&nbsp;├── keystore <br>&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;└── key.pem <br>&nbsp;&nbsp;&nbsp;├── signcerts <br>&nbsp;&nbsp;&nbsp;│&nbsp;&nbsp;&nbsp;└── cert.pem <br>&nbsp;&nbsp;&nbsp;└── user <br></code> | Cannot register or enroll new identities, but transaction traffic does not stop. | https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-tls-ca-s-admin |
+
+
+### Peer Certificates
+
+| Certificate | Description | Location | Impact if expired | Reference |
+|----------------|--------------|---------------------------|------------------|-------------------|
+| Peer Enrollment Certificate <br><br>type=peer | Public key used to make changes to the CA. | Depends on implementation &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br><br><code>org1ca/<br />└── peer1<br />&nbsp;&nbsp;├── msp <br />&nbsp;&nbsp;│   &nbsp;&nbsp;├── cacerts <br />&nbsp;&nbsp;│   &nbsp;&nbsp;│   └── localhost-7053.pem <br />&nbsp;&nbsp;│   &nbsp;&nbsp;├── keystore <br />&nbsp;&nbsp;│   &nbsp;&nbsp;│   └── key.pem <br />&nbsp;&nbsp;│   &nbsp;&nbsp;├── signcerts <br />&nbsp;&nbsp;│   &nbsp;&nbsp;│   └── cert.pem <br />&nbsp;&nbsp;│   &nbsp;&nbsp;└── user <br />&nbsp;&nbsp;└── tls </code> <br />| Production outage <br /><br />Peers do not start without a valid enrollment certificate. | https://hyperledger-fabric-ca.readthedocs.io/en/latest/operations_guide.html#enroll-peer1 |
```

Comment:
(as above concerning the fact that certificate != public key).

Also, the peer enrollment cert doesn't sound like it is used to make changes to the CA. What changes would those be? Doesn't the peer only have one (non-TLS) cert which it uses primarily to produce endorsements?

Reply:
👍 

The Peer Enrollment Certificate is used to assert the identity of the peer node while endorsing transactions.